_opensearch/snapshots/snapshot-management.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: default
+title: Snapshot management
+parent: Snapshots
+nav_order: 20
+has_children: false
+---
+
+# Snapshot management
+
+Snapshot management (SM) is a feature of the [Index Management Plugin]({{site.url}}{{site.baseurl}}/im-plugin) that lets you automate [taking snapshots]({{site.url}}{{site.baseurl}}/opensearch/snapshots/snapshot-restore#take-snapshots). Snapshots have minimal overhead, since they store only incremental changes since the last snapshot. To set up automatic snapshots, you have to create an SM policy with a desired SM schedule and configuration. 
+
+When you create an SM policy, its document ID is given the name `<policy_name>-sm-policy`. Because of this, SM policies have to obey the following rules:
+
+- SM policies must have unique names. 
+
+- You cannot update the policy name after its creation. 
+
+SM-created snapshots have names in the format `<policy_name>-<date>-<random number>`. Two snapshots created by different policies at the same time always have different names because of the `<policy_name>` prefix. To avoid name collisions within the same policy, each snapshot's name contains a hash of time.
+
+Each policy has associated metadata about the status of snapshot creation. The metadata is saved into the system index. The metadata is read before running every scheduled job, so that SM can continue execution from the previous job's state. You can view the metadata using the [explain API]({{site.url}}{{site.baseurl}}/opensearch/snapshots/sm-api#explain).
+
+The cluster’s global metadata must be readable. To include an index in a snapshot, the index and its metadata must also be readable.
+{: .note}
+
+An SM schedule is a custom [cron]({{site.url}}{{site.baseurl}}/monitoring-plugins/alerting/cron) expression that specifies the frequency of taking snapshots, and the timing of creating and deleting snapshots. 
+
+An SM configuration includes the indices and repository for these snapshots, and has all parameters you can define when [creating a snapshot]({{site.url}}{{site.baseurl}}/opensearch/snapshot-restore#take-snapshots) manually. 
+
+An SM configuration can have more than one index.
+{: .note}
+
+## Performance 
+
+One snapshot can contain as many indices as there are in the cluster. We expect at most dozens of SM policies in one cluster, but a snapshot repository can safely scale to thousands of snapshots. However, to manage its metadata, a large repository requires more memory on the cluster manager node. 
+
+Snapshot management depends on the job scheduler plugin to schedule a job that is run periodically. Each SM policy corresponds to one SM scheduled job. The scheduled job is lightweight, so the burden of SM depends on the snapshot creation frequency and the burden of running snapshot operation itself. However, too many pending SM tasks can block other high-priority tasks in the cluster.
```

Comment:
>However, too many pending SM tasks can block other high-priority tasks in the cluster.

what is the remediation if this happens? Pls add

Reply:
> However, too many pending SM tasks can block other high-priority tasks in the cluster.

Can remove this. We are not giving a concept of SM tasks here but only the scheduled job which is lightweight.
And this scheduled job is not blocking other tasks in the cluster.

We are calling out the concurrent running snapshot operation below.

Reply:
Done